pkg/controller/provider/ova-setup.go

```diff
@@ -0,0 +1,205 @@
+package provider
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"strings"
+
+	api "github.com/konveyor/forklift-controller/pkg/apis/forklift/v1beta1"
+	appsv1 "k8s.io/api/apps/v1"
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/intstr"
+)
+
+const (
+	ovaServerPrefix     = "ova-server"
+	ovaImageVar         = "OVA_PROVIDER_SERVER_IMAGE"
+	nfsVolumeNamePrefix = "nfs-volume"
+	mountPath           = "/ova"
+	pvSize              = "1Gi"
+)
+
+func (r *Reconciler) CreateOVAServerDeployment(provider *api.Provider, ctx context.Context) {
+
+	ownerReference := metav1.OwnerReference{
```

Comment:
continuing the comment about the owner above, so you create 2 PVs and 2 PVCs one for the provider and one for the conversion? it might work to use the same one for both (didn't try that)